packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -114,7 +114,8 @@ async function pushToQueue(task, cb) {
     const node = await processRemoteNode(task)
     return cb(null, node)
   } catch (e) {
-    return cb(null, e)
+    console.warn(`Failed to process remote content ${task.url}`)
+    return cb(e)
```

Comment:
This code path seems to never be hit, because of try/catch in `processRemoteNode` function ( https://github.com/gatsbyjs/gatsby/blob/03dc6c97b7ea56ac28de5fd563f3769c74e6e16f/packages/gatsby-source-filesystem/src/create-remote-file-node.js#L262-L265 ).

To me it seems we should remove try/catch there (or rethrow error, but given we don't do anything with error there I think removing is fine)